src/Prefabs/MP/Modes/Roleplay/GameMode_Roleplay.et

```diff
@@ -59,9 +36,6 @@ EL_GameModeRoleplay {
   ScriptedChatEntity ScriptedChatEntity : "{F69BC912AC8236F9}Prefabs/MP/ScriptedChatEntity.et" {
    ID "59751433747F9C02"
   }
-  SCR_GarbageManager GarbageManager {
-   ID "59751433747F9391"
-  }
```

Comment:
For information, this was changed when opening the file after a forced save, I did not manage to keep these lines from the editor.